sst/test/src/tg0_llvm_assemble.cc

```diff
@@ -0,0 +1,13 @@
+#include "sstsimengtest.hh"
+
+TEST_GROUP(TG0, "SSTSimEng_correctly_assembles_instructions_using_LLVM",
+           "fastL1WithParams_config.py", "src",
+           R"(
+    mov x1, #1
+    mov x1, #1
+    mov x1, #1
+  )")
```

Comment:
Fro this test, it may be a good idea to have a few different instructions i.e. mov, arithmetic, fp, simd, sve, ld, str just so we can be sure that all the right extensions are being assembled correctly

Reply:
Although it is copied from regressionTest.hh, I still think it is worth while to ensure that this version of the implementation is done correctly. Plus, it will ensure that passing an instruction string to the CoreWrapper works as expected for a range of instruction types.

Reply:
Please check